hr_contract_multi_job/models/hr_contract_job.py

```diff
@@ -15,15 +15,13 @@ class HrContractJob(models.Model):
     _name = 'hr.contract.job'
     _description = 'Relational object between contract and job'
 
-    name = fields.Char(string='Job', related='job_id.name', index=True)
+    name = fields.Char(string='Job Name', related='job_id.name', index=True)
     job_id = fields.Many2one(
         'hr.job',
-        string='Job',
         required=True,
         ondelete='cascade')
     contract_id = fields.Many2one(
         'hr.contract',
-        string='Contract',
```

Comment:
@luistorresm any particular reason for removing the string?

Reply:
Hello @alexey-pelykh 

Sorry for the time.

In the v12 the string parameter is not necessary. Oddo takes the field name, and if is m2o, removes the `_id` or `_ids`